vm/src/lib.rs

```diff
@@ -30,7 +30,7 @@ extern crate statrs;
 
 extern crate rustpython_parser;
 #[macro_use]
-extern crate rustpython_derive;
+pub extern crate rustpython_derive;
```

Comment:
Why did this have to be made public?

Reply:
If we're using it in `rustpython_vm`, we might as well expose it so that consumers of `rustpython_vm` don't have to add it as a separate dependency. I could do `#[doc(hidden)]`, though.

Reply:
Should we reexport the macros specifically then instead of the crate? (I think that's possible as of rust 2018)